From 14aa3e48600aaeafe36c67d4a1a0e28f2bb3403c Mon Sep 17 00:00:00 2001
From: Weixin Zhou <zwx@rock-chips.com>
Date: Wed, 12 Sep 2018 14:24:58 +0800
Subject: [PATCH] arm64: dts: rockchip: rk3326-863-lp3-v10: modify audio card
 name for vts test

test cmd: run vts -m VtsKernelProcFileApi -t testProcAsoundCardsTest

test result:
shell_command: "cat /proc/asound/cards"
ERROR Failed to parse! Parsing error at token LexToken(COMMA,',',1,48) in line 1
I/ProcessHelper: [VtsKernelProcFileApi]
INFO [Test Case] testProcAsoundCardsTest FAIL

Change-Id: Ib70bf8bb0757a1773253931bb3e6f105eb518126
Signed-off-by: Weixin Zhou <zwx@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi
index 87e6b17de53c..270acbad5e73 100644
--- a/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3326-863-lp3-v10.dtsi
@@ -85,7 +85,7 @@
 	rk817-sound {
 		compatible = "simple-audio-card";
 		simple-audio-card,format = "i2s";
-		simple-audio-card,name = "rockchip,rk817-codec";
+		simple-audio-card,name = "rockchip-rk817-codec";
 		simple-audio-card,mclk-fs = <256>;
 		simple-audio-card,widgets =
 			"Microphone", "Mic Jack",
-- 
2.35.3

